man/runc-resume.8.md

```diff
@@ -9,4 +9,4 @@ resumed.
 
 # DESCRIPTION
    The resume command resumes all processes in the instance of the container.
-Use runc list to identiy instances of containers and their current status.
+Use runc list to identity instances of containers and their current status.
```

Comment:
seems like all instances of this should be `identify`

Reply:
Thanks for catching that!